source/components/organisms/Step/Step.js

```diff
@@ -219,6 +222,7 @@ Step.propTypes = {
    * The array of fields that are going to be displayed in the Step
    */
   questions: PropTypes.arrayOf(PropTypes.object),
+  allQuestions: PropTypes.array,
```

Comment:
What's the difference between questions and allQuestions?

Reply:
questions is the questions belonging to the step. allQuestions is all questions in the form.